src/electionguard_gui/services/key_ceremony_service.py

```diff
@@ -179,3 +174,11 @@ def set_complete(
             {"_id": ObjectId(key_ceremony_id)},
             {"$set": {"completed_at": datetime.utcnow()}},
         )
+
+    def get_completed(self, db: Database) -> List[KeyCeremonyDto]:
+        key_ceremonies = db.key_ceremonies.find({"completed_at": {"$exists": True}})
```

Comment:
This is a minor bug, `completed_at` will always exist, this should instead check for null.